docs/accordproject-faq.md

```diff
@@ -17,42 +17,41 @@ While the Accord Project technology is targeted at the development of smart lega
 
 ### How is an Accord Project "Smart Contract" different from "Smart Contracts" on the blockchain?
 
-Accord Project Smart legal contracts should not be confused with so-called blockchain “smart contracts”, which are scripts that necesarily operate on a blockchain. On the blockchain a smart contract is often written in a specific language like solidity that executes and operates on the blockchain. It lives in a closed world. An Accord Project Smart Contract contains text based template that integrates with a data model and the Ergo language. The three components are integrated into a whole. Using Ergo an Accord Project Smart contract can communicate with other systems, it can send and receive data, it can perform calculations and it can interact with a blockchain.
+Accord Project Smart legal contracts should not be confused with so-called blockchain “smart contracts”, which are scripts that necessarily operate on a blockchain. On the blockchain, a smart contract is often written in a specific language like 'solidity' that executes and operates on the blockchain most notably used by Ethereum. Ethereum being a major platform for generating smart contracts, other blockchains also create contracts compatible with Ethereum-based ones so that they can be portable on the Ethereum network as well. It lives in a closed world. An Accord Project Smart Contract contains text based template that integrates with a data model and the Ergo language. The three components are integrated into a whole. Using Ergo an Accord Project Smart contract can communicate with other systems, it can send and receive data, it can perform calculations and it can interact with a blockchain making it highly versatile.
```

Comment:
I think ` most notably used by Ethereum. Ethereum being a major platform for generating smart contracts, other blockchains also create contracts compatible with Ethereum-based ones so that they can be portable on the Ethereum network as well. ` should be removed.

Reply:
the documentation as is had the statement, "a smart contract is often written in a specific language like solidity that executes and operates on the blockchain". the language of this statement sort of confused me at first glance and I had to read the sentence thrice before the meaning was conveyed. so I thought I would add a bit of context to it so it was easier to understand. Ethereum came in cause of the mention of the word "solidity" which implied more or less the same. My intention was to let people know, about being able to be cross-compatible with the biggest blockchain dealing with smart-contracts, but I guess the phrasing was aggressive. I definitely understand @dselman and highly agree with it. I just thought I would provide a different perspective. Thanks.